package.json

```diff
@@ -31,6 +31,7 @@
     "react-scripts-ts": "2.15.1",
     "redux": "3.7.2",
     "redux-mock-store": "1.5.1",
+    "redux-persist": "^5.9.1",
```

Comment:
Oh one last thing. Can we change this to fixed versions?

Reply:
Fixed to 5.9.1